js/package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Protocol Buffers for JavaScript",
   "main": "google-protobuf.js",
   "dependencies": {
-    "google-closure-library": "~20160125.0.0",
```

Comment:
<receiver android:name=".app.DeviceAdminSample$DeviceAdminSampleReceiver"
        android:label="@string/sample_device_admin"
        android:description="@string/sample_device_admin_description"
        android:permission="android.permission.BIND_DEVICE_ADMIN">
    <meta-data android:name="android.app.device_admin"
               android:resource="@xml/device_admin_sample" />
    <intent-filter>
        <action android:name="android.app.action.DEVICE_ADMIN_ENABLED" />
    </intent-filter>
</receiver>
<receiver android:name=".app.DeviceAdminSample$DeviceAdminSampleReceiver2"
    android:label="@string/sample_device_admin2"
    android:description="@string/sample_device_admin_description2"
    android:permission="android.permission.BIND_DEVICE_ADMIN">
    <meta-data android:name="android.app.device_admin"
        android:resource="@xml/device_admin_sample" />
    <intent-filter>
        <action android:name="android.app.action.DEVICE_ADMIN_ENABLED" />
    </intent-filter>
</receiver>
The meta-data referenced here provides addition information specific to the device administrator, as parsed by the DeviceAdminInfo class. A typical file would be:

<device-admin xmlns:android="http://schemas.android.com/apk/res/android">
    <uses-policies>
        <limit-password />
        <watch-login />
        <reset-password />
        <force-lock />
        <wipe-data />
        <expire-password />
        <encrypted-storage />
        <disable-camera />
        <disable-keyguard-features />
    </uses-policies>
</device-admin>
Developer Guides
For more information about device administration, read the Device Administration developer guide.

Summary
Constants

















































Public constructors
Public methods








































Newsletter Blog Support
ANDROID DEVELOPERS ON YOUTUBE ANDROID DEVELOPERS ON GOOGLE+ ANDROID DEVELOPERS ON TWITTER
Except as noted, this content is licensed under Creative Commons Attribution 2.5. For details and restrictions, see the Content License.
Android.com Android Developers Android Studio Google Play Developer Console Legal N